### PR TITLE
add support for Vertica

### PR DESCRIPTION
**Pull request by baron42bba**
Added support for https://www.vertica.com

Signed-off-by: Andreas Gerler <baron@bundesbrandschatzamt.de>

### CODE REVIEW

**Comment by mattn** [comment]
please rebase master branch.

**Comment by baron42bba** [comment]
Hi @mattn !
Thanks for taking care of the codebase.
I rebased today.

File: README.md [hunk 1]
```diff
@@ -123,6 +124,9 @@ connections:
       user: sshuser
       passPhrase: ssspass
       privateKey: /home/sqls-server/.ssh/id_rsa
+  - alias: dsn_vertica
+    driver: vertica
+	dataSourceName: vertica://user:pass@host:5433/dbname
```
Comment (mattn) [inline README.md:129]: different indentation?

Comment (baron42bba) [reply README.md]: untabified :-)

File: internal/database/vertica.go [hunk 2]
```diff
@@ -0,0 +1,237 @@
+package database
+
+import (
+	"context"
+	"database/sql"
+	"github.com/lighttiger2505/sqls/dialect"
```
Comment (mattn) [inline internal/database/vertica.go:6]:
```suggestion
	"github.com/sqls-server/sqls/dialect"
```

**Comment by mattn** [comment]
Still build failed. missing DescribeForeignKeysBySchema

```go
func (db *VerticaDBRepository) DescribeForeignKeysBySchema(ctx context.Context, schemaName string) ([]*ForeignKey, error) {
    //
}
```

**Comment by mattn** [comment]
Thank you

**Pull request merged**